index.ts

```diff
@@ -138,17 +138,19 @@ function $$<Selected extends Element>(
 	// Can be: select.all('selectors') or select.all('selectors', singleElementOrDocument)
 	if (!baseElements || isQueryable(baseElements)) {
 		const elements = (baseElements ?? document).querySelectorAll<Selected>(String(selectors));
-		return Array.prototype.slice.call(elements) as Selected[];
+		return Array.from(elements) as Selected[];
```

Comment:
I’d also prefer this, but it’s actually slower: https://www.measurethat.net/Benchmarks/Show/12088/3/new-array-from-vs-slice-vs-push-vs-index-vs-spread